lib/questions.rb

```diff
@@ -147,34 +182,41 @@ def format_date_nicely(date)
 # get the domain name *without* the .com part, from an email address
 # so alex@makersacademy.com becomes makersacademy
 def get_domain_name_from_email_address(email)
+  domain = email.gsub(/.+@([^.]+).+/, '\1')
 end
 
-# capitalize the first letter in each word of a string, 
+# capitalize the first letter in each word of a string,
 #  except 'a', 'and' and 'the'
 # *unless* they come at the start of the start of the string, e.g.
 # 'the lion the witch and the wardrobe' becomes
 # 'The Lion the Witch and the Wardrobe'
 def titleize_a_string(string)
+  title_cases = Hash.new {|_,k| k.capitalize }.merge({'And' => 'and', 'The' => 'the', 'A' => 'a'})
```

Comment:
Line is too long. [98/80]<br>Redundant curly braces around a hash parameter.<br>Space missing after comma.<br>Space between { and | missing.<br>Space inside { missing.<br>Space inside } missing.